sampler.go

```diff
@@ -34,8 +34,8 @@ const defaultSamplingServerHostPort = "localhost:5778"
 
 // Sampler decides whether a new trace should be sampled or not.
 type Sampler interface {
-	// IsSampled decides whether a trace with given `id` should be sampled.
-	IsSampled(id uint64) bool
+	// IsSampled decides whether a trace with given `id` and `operation` should be sampled.
+	IsSampled(id uint64, operation string) bool
```

Comment:
I was thinking that this would need to return `(sampled bool, tags []tag)`, otherwise you cannot correlate a given decision with the correct tags.